string idle_robot
---
string response